pkgs/by-name/cp/cplay-ng/package.nix

```diff
@@ -1,32 +1,27 @@
 {
   lib,
-  python3,
+  python3Packages,
   fetchFromGitHub,
   makeWrapper,
   mpv,
   pulseaudio,
 }:
 
-python3.pkgs.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
   pname = "cplay-ng";
-  version = "5.3.1";
+  version = "5.4.0";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "xi";
     repo = "cplay-ng";
-    rev = version;
-    hash = "sha256-6mphhoLDkGZ2r+elzLlPl3B8fNz3loqrQB8x8276AHI=";
+    tag = version;
+    hash = "sha256-ob5wX+Q5XKB/fTYG5phLU61imonpk2A/fk5cg/dfr1Y=";
   };
 
-  nativeBuildInputs = [
-    makeWrapper
-  ];
-
-  build-system = [
-    python3.pkgs.setuptools
-  ];
+  nativeBuildInputs = [ makeWrapper ];
 
-  pyproject = true;
+  build-system = [ python3Packages.setuptools ];
```

Comment:
As far as I can see `pyproject` should remain `true`

https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters
> As all other formats are deprecated, you are recommended to set [pyproject] to true.

Reply:
@emaryn you marked the conversation as resolved, but you didn't comment or push anything, so I unresolved it

Reply:
Oh sorry I didn't notice it at first